docs-source/content/security/openshift.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "OpenShift"
+date: 2019-10-04T08:08:08-05:00
+weight: 7
+description: "OpenShift information for the WebLogic operator"
+---
+
+#### OpenShift `anyuid` security context
+
+The Docker images that Oracle publishes default the container user
+as `oracle` which is UID `1000` and GID `1000`. When running the
+Oracle images or layered images that retain the default user as
+`oracle` with OpenShift, the `anyuid` security context constraint
+is required to ensure proper access to file system within the
+Docker image. This means that the administrator must:
+
+1. Ensure the `anyuid` security content is granted
+2. Ensure that WebLogic containers are annotated with `openshift.io/scc: anyuid`
+
+For example, to update the OpenShift policy use:
+
+```bash
+$ oc adm policy add-scc-to-user anyuid -z default
+```
+
+and to annotate the WebLogic cotnainers, update the WebLogic `Domain` resource
+to include `annotations` for the `serverPod`. For example:
+
+``` yaml
+kind: Domain
+metadata:
+  name: domain1
+spec:
+  domainUID: domain1
+  serverPod:
+    env:
+      - name: var1
+        value: value1
+    annotations: 
+      openshift.io/scc: anyuid
+```
+
+{{% notice note %}}
+For additional information about OpenShift requirements and the WebLogic operator,
+see the [OpenShift]({{<relref  "/userguide/introduction/introduction.md#openshift">}}) section in the User Guide.
```

Comment:
Add a corresponding link to this new section in the reverse direction (from the introduction)?

Reply:
You people are so picky, no wonder why no one wants to take on documentation tasks! 😃

Reply:
Done.